app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    rename_case_flow = command.split(" ")
```

Comment:
rename this variable.
<img width="379" alt="image" src="https://github.com/mate-academy/py-move-file/assets/65166203/6231b197-158c-415d-bf54-4ce0ab3d8371">

Rename it to command_parameters